_posts/2022-03-22-Introducing-Data-Prepper-1.3.0-with-New-Aggregation-Processor.md

```diff
@@ -0,0 +1,224 @@
+amount of time. Depending on your logs, the aggregate action being used, and the amount of memory available, the 
+aggregation could take place over a long period of time.
+
+In Data Prepper 1.3, two actions are being released with the `aggregate` processor: `remove_duplicates` and `put_all`. 
+However, creating custom actions is extremely simple. If you are interested in learning more about creating custom actions, read the 
+[aggregate README](https://github.com/opensearch-project/data-prepper/tree/main/data-prepper-plugins/aggregate-processor#creating-new-aggregate-actions). 
+Additionally, please create a [Github issue](https://github.com/opensearch-project/data-prepper/issues/new/choose) 
+for any actions you would like Data Prepper to support.
+
+At the moment, the aggregate processor is only useful for single-node clusters of Data Prepper. However, utilizing the 
+[peer forwarder](https://github.com/opensearch-project/data-prepper/issues/700)
+to aggregate over multiple-node clusters is planned for a future release of Data Prepper.
+
+The following pipeline configuration extracts fields of `sourceIp`, `destinationIp`, and `port` using the `grok`
+processor, and then aggregates on those fields over a period of 30 seconds using the `aggregate` processor and 
+the `put_all` action. At the end of the 30 seconds, the aggregated log is sent to the OpenSearch sink.
+
+```
+aggregate_pipeline:  
+   source:
+     http:
+       ssl: false
+   processor:
+     - grok:
+         match: 
+           log: ["%{IPORHOST:sourceIp} %{IPORHOST:destinationIp} %{NUMBER:port:int}"]
+          
+     - aggregate:
+         group_duration: "30s"
+         identification_keys: ["sourceIp", "destinationIp", "port"]
+         action:
+           put_all:
+   sink:
+     - opensearch:
+         hosts: ["https://opensearch:9200"]
+         index: aggregated_logs
+```
+
+Given the following batch of logs:
+
+```
+  { "log": "127.0.0.1 192.168.0.1 80", "status": 200 }
+  { "log": "127.0.0.1 192.168.0.1 80", "bytes": 1000 }
+  { "log": "127.0.0.1 192.168.0.1 80" "http_verb": "GET" }
+```
+
+The grok processor will extract the `identification_keys` to create the following logs:
+
+```
+  { "sourceIp": "127.0.0.1", "destinationIp": "192.168.0.1", "port": 80, "status": 200 }
+  { "sourceIp": "127.0.0.1", "destinationIp": "192.168.0.1", "port": 80, "bytes": 1000 }
+  { "sourceIp": "127.0.0.1", "destinationIp": "192.168.0.1", "port": 80, "http_verb": "GET" }
+```
+
+And when the group is concluded after a duration of 30 seconds from the time that the first log is 
+received by the `aggregate` processor, the following aggregated log will be shipped to the sink:
+
+```
+{ "sourceIp": "127.0.0.1", "destinationIp": "192.168.0.1", "port": 80, "status": 200, "bytes": 1000, "http_verb": "GET" }
+```
+
+## Other Improvements
+
+In addition to the new features already described, we have made a few other improvements to Data Prepper.
```

Comment:
I'd revise this - does 'we' refer to the authors of the blog post or the software?